client/config/index.js

```diff
@@ -0,0 +1,11 @@
+let config;
+
+if ( typeof window !== 'undefined' && window.config ) {
+	config = window.config;
+	config.anyEnabled = window.anyEnabled;
+	config.isEnabled = window.isEnabled;
+} else {
+	throw new Error( 'config not available' );
+}
```

Comment:
> window.config is a function here. I don't think spread works for those

Just tested it out in the online babel repl

```jsx
a = () => 5;
b = { ...a }

console.log( a() );
console.log( b() );
```

transpiled to:
```jsx
"use strict";

var _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; };

a = function a() {
  return 5;
};
b = _extends({}, a);

console.log(a());
console.log(b());
```

which printed out:
```
Uncaught TypeError: b is not a function
    at <anonymous>:1:1
```